test/join.jl

```diff
@@ -578,4 +578,23 @@ module TestJoin
         @test eltypes(join(l, r, on=:b, kind=:outer, makeunique=true)) ==
             [Union{Int, Missing}, CS, Union{Int, Missing}]
     end
+
+    @testset "test checks of merge key uniqueness" begin
+
+        @test_throws ArgumentError join(name, job, on=:ID,
+                     check_uniqueness=(false, true))
```

Comment:
Align arguments with parentheses (or maybe they even fit on the previous line?).